middle_end/flambda2/simplify/simplify_let_cont_expr.ml

```diff
@@ -919,63 +1069,98 @@ let sort_handlers data handlers =
       group :: inner)
     [] sorted_handlers_from_the_inside_to_the_outside
 
-let after_downwards_traversal_of_body_and_handlers
-    (data : after_downwards_traversal_of_body_and_handlers_data) ~down_to_up
-    dacc =
+let rec after_downwards_traversal_of_body_and_handlers ~simplify_expr
```

Comment:
All those functions are now recursive. I think it is not required. That's good for review that they didn't move. But reordering them and removing the rec might be better afterwards.
```suggestion
let rec after_downwards_traversal_of_body_and_handlers ~simplify_expr
(* CR pchambart: Those functions don't need to be recursive, it would be nice to fix that later *)
```